apps/api/src/dotenvcreate.mjs

```diff
@@ -0,0 +1,105 @@
+import { SecretsManagerClient, GetSecretValueCommand } from '@aws-sdk/client-secrets-manager';
```

Comment:
This script needs to be used across all services, so it should be stored outside the API.

Reply:
This we will discuss in sync